src/components/Collapser.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600&display=swap');
```

Comment:
Since the CSS is all in emotion now, you should be able to just delete this file.

Reply:
oops, thought i did 😬